packaging/increment_build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e # Fail on error
```

Comment:
I like to do `set -e -u -o pipefail` for extra bulletproofiness. The `-u` means that we'd have to replace any `$MAYBE_DEFINED_VAR` with `${MAYBE_DEFINED_VAR:-}`, though I don't think I see any cases like that.